### ISSUE TITLE
There is no Azure Repos Plugin for IntelliJ

### ISSUE THREAD

**Issue by c031917** [MicrosoftDocs/azure-devops-docs#9684]
[Enter feedback here]
document talks about installing Azure Repos Plugin for IntelliJ, but this does not exist. Do you mean Azure DevOps Plugin ?
Does that plugin work the same way ?
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 92d64059-6cb2-1927-d751-68239d7084d6
* Version Independent ID: 5cb8443e-14dd-fe98-570c-f5245fc6ca90
* Content: [Azure Repos Plugin for IntelliJ, IDEA, Android Studio - Azure Repos](https://docs.microsoft.com/en-us/azure/devops/repos/git/create-repo-intellij?view=azure-devops)
* Content Source: [docs/repos/git/create-repo-intellij.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/repos/git/create-repo-intellij.md)
* Product: **devops**
* Technology: **devops-code-git**
* GitHub Login: @vtbassmatt
* Microsoft Alias: **macoope**

**Comment by vtbassmatt** [comment]
True. We should link out to [this](https://plugins.jetbrains.com/plugin/7981-azure-devops) and it's indeed called "Azure DevOps", not "Azure Repos".
